examples/window/window_drag_move.rs

```diff
@@ -0,0 +1,140 @@
+
+/// Directions that the drag resizes the window toward.
+const DIRECTIONS: [CompassOctant; 8] = [
+    CompassOctant::North,
+    CompassOctant::NorthEast,
+    CompassOctant::East,
+    CompassOctant::SouthEast,
+    CompassOctant::South,
+    CompassOctant::SouthWest,
+    CompassOctant::West,
+    CompassOctant::NorthWest,
+];
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(WindowPlugin {
+            primary_window: Some(Window {
+                decorations: false,
+                ..default()
+            }),
+            ..default()
+        }))
+        .insert_resource(ResizeDir(7))
+        .insert_resource(LeftClickAction::Drag)
+        .add_systems(Startup, setup)
+        .add_systems(Update, (handle_input, move_windows))
+        .run();
+}
+
+fn setup(mut commands: Commands) {
+    // Camera
+    commands.spawn(Camera3d::default());
+
+    // UI
+    commands
+        .spawn((
+            NodeBundle {
+                style: Style {
+                    position_type: PositionType::Absolute,
+                    padding: UiRect::all(Val::Px(5.0)),
+                    ..default()
+                },
+                background_color: Color::BLACK.with_alpha(0.75).into(),
+                ..default()
+            },
+            GlobalZIndex(i32::MAX),
+        ))
+        .with_children(|p| {
+            p.spawn(Text::default()).with_children(|p| {
+                p.spawn(TextSpan::new(
+                    "Demonstrate drag move and drag resize without window decorations.\n\n",
+                ));
+                p.spawn(TextSpan::new("Controls:\n"));
+                p.spawn(TextSpan::new("A - change left click action ["));
+                p.spawn(TextSpan::new("Drag"));
+                p.spawn(TextSpan::new("]\n"));
+                p.spawn(TextSpan::new("S / D - change resize direction ["));
+                p.spawn(TextSpan::new("NorthWest"));
+                p.spawn(TextSpan::new("]\n"));
+            });
+        });
+}
+
+fn handle_input(
+    input: Res<ButtonInput<KeyCode>>,
+    mut action: ResMut<LeftClickAction>,
+    mut dir: ResMut<ResizeDir>,
+    example_text: Query<Entity, With<Text>>,
+    mut writer: UiTextWriter,
+) {
+    use LeftClickAction::*;
+    if input.just_pressed(KeyCode::KeyA) {
+        *action = match *action {
+            Drag => Resize,
+            Resize => Nothing,
+            Nothing => Drag,
+        };
+        *writer.text(example_text.single(), 4) = format!("{:?}", *action);
+    }
+
+    if input.just_pressed(KeyCode::KeyS) {
+        dir.0 = dir
+            .0
+            .checked_sub(1)
+            .unwrap_or(DIRECTIONS.len().saturating_sub(1));
+        *writer.text(example_text.single(), 7) = format!("{:?}", DIRECTIONS[dir.0]);
+    }
+
+    if input.just_pressed(KeyCode::KeyD) {
+        dir.0 = (dir.0 + 1) % DIRECTIONS.len();
+        *writer.text(example_text.single(), 7) = format!("{:?}", DIRECTIONS[dir.0]);
+    }
+}
+
+fn move_windows(
```

Comment:
```suggestion
fn drag_or_resize_window(
```
I feel like move only refers to drag